<launch>
    <node name="piksi_multi_cpp_base" pkg="piksi_multi_cpp" type="piksi_multi" output="screen">
        <param name="device_ids" value="dummy://test" />

        <param name="unknown_receiver_0/udp_port"
               value="26078"/> <!-- Port on which UDP broadcasts are sent -->

        <param name="unknown_receiver_0/udp_broadcast_interfaces"
               value="lo;enp0s31f6:"/> <!-- Semicolon separate list of interfaces on which observations should be sent via network broadcast -->

        <param name="unknown_receiver_0/udp_unicast_targets"
               value=""/> <!-- Additional hosts to which observations are sent via unicast -->
    </node>
</launch>